app/components/post_list/post/body/reactions/reaction.tsx

```diff
@@ -86,7 +88,7 @@ const Reaction = ({count, emojiName, highlight, onPress, onLongPress, theme}: Re
             <View style={styles.countContainer}>
                 <AnimatedNumbers
                     includeComma={false}
-                    fontStyle={[styles.count, (highlight && styles.countHighlight)]}
+                    fontStyle={fontStyle}
```

Comment:
Memoizing this we remove some unneded re-renderings. Not a big deal (~1ms) but also cleans up the profiling.